realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -1304,20 +1458,22 @@ void BaseRealSenseNode::updateStreamCalibData(const rs2::video_stream_profile& v
             {
                 auto video_profile = profile.as<rs2::video_stream_profile>();
                 stream_index_pair stream_index{video_profile.stream_type(), video_profile.stream_index()};
-                _depth_aligned_camera_info[stream_index] = _camera_info[DEPTH];
+                _depth_aligned_camera_info[stream_index] = _camera_info[stream_index];
```

Comment:
@dantwinkler can you verify that this line is correct? I recall that you were looking at these mappings in the past